Source/WebKit/Platform/SharedMemory.h

```diff
@@ -69,25 +80,19 @@ class SharedMemoryHandle {
 
     void clear();
 
-    void encode(IPC::Encoder&) const;
-    static WARN_UNUSED_RETURN bool decode(IPC::Decoder&, SharedMemoryHandle&);
 #if USE(UNIX_DOMAIN_SOCKETS)
     UnixFileDescriptor releaseHandle();
 #endif
 
 private:
-#if USE(UNIX_DOMAIN_SOCKETS)
-    mutable UnixFileDescriptor m_handle;
-#elif OS(DARWIN)
-    mutable MachSendRight m_handle;
-#elif OS(WINDOWS)
-    mutable Win32Handle m_handle;
-#endif
```

Comment:
@darinadler the `m_handle` is marked as `mutable` so the `WTFMove` is valid in the encoder. At this time I'm not able to get rid of the `mutable` but plan to continue on and try to remove it in the future.